src/nupic/algorithms/Cells4.hpp

```diff
@@ -406,8 +407,8 @@ namespace nupic {
                Real permInc =.1,
                Real globalDecay =0,
                bool doPooling =false,
-               int seed =-1,
-               bool doItAll =false,
+               int seed =42,
```

Comment:
If we change it then I need to go check a bunch of stuff to make sure it doesn't impact other places. So can we move that to a PR dedicated to changing the defaults? I'd like to get this PR mixed without that change first.